src/mbgl/map/transform.cpp

```diff
@@ -438,12 +438,14 @@ void Transform::setScale(double scale, optional<EdgeInsets> padding, const Anima
     setScale(scale, anchor, animation);
 }
 
-void Transform::setMinZoom(const double minZoom) {
+#pragma mark - Bounds
+
+void Transform::setMinZoom(double minZoom) {
     if (std::isnan(minZoom)) return;
     state.setMinZoom(minZoom);
 }
 
-void Transform::setMaxZoom(const double maxZoom) {
+void Transform::setMaxZoom(double maxZoom) {
```

Comment:
As you mentioned, there is no point in adding this to headers. Also, _const-pass-by-value_ is not a common behavior in our codebase.

For me, it is clear that a non-const _pass-by-value_ is a different object from the one it has been copied from i.e. declaring it `const` does not guarantee that the original object remains immutable.